roles/custom/matrix-bot-draupnir/templates/production.yaml.j2

```diff
@@ -251,10 +287,13 @@ web:
     enabled: {{ matrix_bot_draupnir_abuse_reporting_enabled | to_json }}
 {% endif %}
 
+# FIXME: This configuration option is currently broken in the playbook as admin APIs can not
+# be accessed from containers. See https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/3389
+# and https://github.com/spantaleev/matrix-docker-ansible-deploy/issues/3308
 # Whether or not to actively poll synapse for abuse reports, to be used
 # instead of intercepting client calls to synapse's abuse endpoint, when that
 # isn't possible/practical.
-pollReports: false
+#pollReports: false
```

Comment:
Is this intentional?

Reply:
Yes. its commented out due to the fact it doesnt work.